app/src/pages/admin/projectStatusPage/statusPageContent/widgets/activityPanel/activityPanel.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { ProjectActivity } from 'components/widgets/tables/projectActivity';
+import styles from './activityPanel.scss';
+
+const cx = classNames.bind(styles);
+
+const CONTENT_FIELDS = [
+  'history',
+  'loggedObjectRef',
+  'projectRef',
+  'objectType',
+  'actionType',
+  'lastModified',
+  'user',
+  'name',
+];
+
+const ACTION_TYPE = [
+  'startLaunch',
+  'finishLaunch',
+  'deleteLaunch',
+  'postIssue',
+  'linkIssue',
+  'unlinkIssue',
+  'createUser',
+  'createDashboard',
+  'updateDashboard',
+  'deleteDashboard',
+  'createWidget',
+  'updateWidget',
+  'deleteWidget',
+  'createFilter',
+  'updateFilter',
+  'deleteFilter',
+  'createBts',
+  'updateBts',
+  'deleteBts',
+  'updateProject',
+  'updateAnalyzer',
+  'generateIndex',
+  'deleteIndex',
+  'updateDefect',
+  'deleteDefect',
+  'startImport',
+  'finishImport',
+];
+
+const WIDGET_TYPE = 'activityStream';
+
+export const ActivityPanel = ({ data: { result } }) => {
+  const widget = {
+    content: { result: result.slice(0, 150) },
```

Comment:
Is it requirement?

Reply:
Yes. "Widget shows last 150 activities occurring on the project"

Reply:
So shouldn't this be controlled by backend side? Should the UI side receive unnecessary activities?